### ISSUE TITLE
Immutable surveys

### ISSUE THREAD

**Issue by markus2120** [simonv3/quick-survey#62]
Would it be possible to add a immutable option so that the survey cannot be reset? I want this as a feedback/report system for a hospital. Thanks for the help.

**Comment by simonv3** [comment]
Are you looking for a way for admin to say whether the "reset" button can be clicked by non-admin (ie. moderators, or whatever I've called them)? 

Also, I would love to hear a bit more about the setting you're using this in.

**Comment by markus2120** [comment]
No, I meant once the application/survey is created it must be having the submissions non-deletable. We plan to use it as anonymous system for a hospital to list all suggestions or complaints (from employees). That way even the admin cannot reset submitted data. This will give *confidence* to our employees that the suggestion/complaint was NOT deleted by a unfair or bad admin.

**Comment by simonv3** [comment]
Ah, that's kind of cool. Like I said in another ticket - I don't have much time to put into this, but I'm definitely open to PRs.
